azure-kusto-ingest/azure/kusto/ingest/ingestion_properties.py

```diff
@@ -218,6 +220,8 @@ def __init__(
         self.report_method = report_method
         self.validation_policy = validation_policy
         self.additional_properties = additional_properties
+        self.application_for_tracing = application_for_tracing
```

Comment:
Unless I'm missing something, you don't forward it to IngestionBlobInfo, so this goes nowhere

Reply:
Why not copy ctoring it ?

Reply:
see,s like ingestionblobinfo handles ingestionproperties field differently for each field, so not sure we just wanna copy it